runtime/cere-dev/src/lib.rs

```diff
@@ -958,6 +979,8 @@ impl pallet_contracts::Config for Runtime {
 	type Schedule = Schedule;
 	type AddressGenerator = pallet_contracts::DefaultAddressGenerator;
 	type ContractAccessWeight = pallet_contracts::DefaultContractAccessWeight<RuntimeBlockWeights>;
+	type MaxCodeLen = ConstU32<{ 128 * 1024 }>;
```

Comment:
Why `128`?

Reply:
Could you please check it during the test run?

Reply:
Tested. Works as expected.